sonar-batch/src/main/java/org/sonar/batch/bootstrap/GlobalContainer.java

```diff
@@ -19,14 +19,16 @@
  */
 package org.sonar.batch.bootstrap;
 
+import org.sonar.batch.index.CachesManager;
```

Comment:
Are you sure your Eclipse configuration is fine with coding guidelines? This import part looks strange.